sqoop/README.MD

```diff
@@ -0,0 +1,74 @@
+# Sqoop 
+
+This initialization action installs [Sqoop](http://sqoop.apache.org/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. Sqoop is being compiled from trunk branch of official [github](https://github.com/apache/sqoop) mirror.
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with Sqoop installed by:
+
+1. Uploading a copy of this initialization action (`sqoop.sh`) to [Google Cloud Storage](https://cloud.google.com/storage).
```

Comment:
Sorry I haven't merged my README updates, but I'm planning to change the READMEs to just point at our GCS mirror: https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/262.

Please follow the style from that PR :)

Reply:
ok